elements/pl-matrix-output/pl-matrix-output.py

```diff
@@ -6,19 +6,45 @@
 
 def prepare(element_html, data):
     element = lxml.html.fragment_fromstring(element_html)
-    pl.check_attribs(element, required_attribs=[], optional_attribs=['digits'])
+    required_attribs = []
+    optional_attribs = ['digits', 'default-tab', 'show-matlab', 'show-mathematica', 'show-python']
+    pl.check_attribs(element, required_attribs, optional_attribs)
 
 
 def render(element_html, data):
     element = lxml.html.fragment_fromstring(element_html)
     digits = pl.get_integer_attrib(element, 'digits', 2)
+    display_matlab_tab = pl.get_boolean_attrib(element, 'show-matlab', True)
+    display_mathematica_tab = pl.get_boolean_attrib(element, 'show-mathematica', True)
+    display_python_tab = pl.get_boolean_attrib(element, 'show-python', True)
+    default_tab = pl.get_string_attrib(element, 'default-tab', 'matlab')
 
+    # Handles cases when Matlab tab hidden, if python is default tab don't overwite active tab setting
+    if display_matlab_tab is False and default_tab != 'python':
```

Comment:
I think this code will work, but it seems weird. I think it’s trying to say:
```
if not display_matlab_tab and default_tab == ‘matlab’:
```

Reply:
There is a block of code after this that sets the default tab to be the active tab. 

This code block is meant to set the first existing tab to be active. So if there is a Mathematica and Python tab, Mathematica will be the active tab. However, if the user specifies that Python should be the active tab, it should skip this code block.